### PR TITLE
kops-ci: Use container image for boostrap

### PR DESCRIPTION
**Pull request by ameukam**
Follow-up of:
  - https://github.com/kubernetes/k8s.io/pull/6148

Use a custom image for bootstrap since Bottlerocket only TOML format for bootstrap

### CODE REVIEW

File: infra/aws/terraform/kops-infra-ci/resources/bootstrap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+FROM alpine@sha256:34871e7290500828b39e22294660bee86d966bc0017544e848dd9a255cdf59e0
+
+RUN apk update \
+    && apk add --no-cache \
+        bash \
+        findutils \
+        util-linux \
+        xfsprogs
+
+ADD https://raw.githubusercontent.com/kubernetes/k8s.io/a8be6bcc88c61dc55ce0f6b888e7f4e6c0a5293d/infra/aws/terraform/prow-build-cluster/bootstrap_bottlerocket/node_bootstrap.sh /bootstrap.sh
```
Comment (dims) [inline infra/aws/terraform/kops-infra-ci/resources/bootstrap/Dockerfile:26]: this this make it harder to update bootstrap.sh in the future? (update the script in one PR and then fix up the SHA here in the next one?)

Comment (ameukam) [reply infra/aws/terraform/kops-infra-ci/resources/bootstrap/Dockerfile:26]: Assuming we are doing a lot of changes for this. We are not gonna change the instance type for a while (specially in we use 1-year) and the script is pretty stable. 
Happy to use the ref of a branch if you want.

Comment (bryantbiggs) [reply infra/aws/terraform/kops-infra-ci/resources/bootstrap/Dockerfile:26]: what about https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/getting-started.md#option-5-eks-experimental instead?

**Comment by ameukam** [comment]
Ref:
 - https://github.com/kubernetes/k8s.io/issues/5127

**Pull request merged**